src/Files.App/Utils/Taskbar/SystemTrayIcon.cs

```diff
@@ -27,7 +27,19 @@ public sealed class SystemTrayIcon : IDisposable
 
 		// Fields
 
-		private readonly static Guid _trayIconGuid = new("684F2832-AC2B-4630-98C2-73D6AEBD46B7");
+		private readonly static Guid _trayIconGuid = new(
```

Comment:
Are these random values? Can they be a little more specific?

Reply:
Sure, some COM contract GUIDs that are associated with the same program uses almost the same GUID but the last part.

For example, `684F2832-AC2B-4630-98C2-000000000001`.
I'm not sure this would break something.

Reply:
Does they need to be numbers?

Reply:
Has to be a GUID since this is used by a native API to distinguish the existing tray icons and avoid creating the same icon.

Reply:
In that case I would follow your suggestion to use similar GUIDs.